static/js/components/tiles/answer_table_tile.tsx

```diff
@@ -66,6 +70,10 @@ export function AnswerTableTile(props: AnswerTableTilePropType): JSX.Element {
     >
       <div className={`answer-table-content ${ASYNC_ELEMENT_CLASS}`}>
         {props.title && <div className="answer-table-title">{props.title}</div>}
+        <DownloadButton
+          content={answerAsCsv}
+          filename={`${props.title}.csv`}
```

Comment:
For the query "how many species of the flu are there?" I'm getting a file download named "undefined.csv"